src/pages/home_page/HomePage.jsx

```diff
@@ -0,0 +1,123 @@
+import "./home_page.css";
+import collection_img from "./../../assets/images/collection_item_img.jpeg"
+import { ButtonPrimary } from "../../components/Button";
+
+const Categories = ({ link, category, img_url }) => {
+  return (
+    <a href={link}>
+      <div class="categories">
+        <img src={img_url} />
```

Comment:
Suggestion: provide "alt" in < img /> tag, it will help in accessibility

Reply:
ok i will add that, thank you for pointing out